components/discord_webhook/actions/send-message-with-file/send-message-with-file.js

```diff
@@ -0,0 +1,73 @@
+const discordWebhook = require("../../discord_webhook.app.js");
+var request = require("request");
+const fs = require("fs");
+
+module.exports = {
+  key: "discord_webhook-send-message-with-file",
+  name: "Send Message With File",
+  description: "Post a message with an attached file",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    discordWebhook,
+    message: {
+      propDefinition: [
+        discordWebhook,
+        "message",
+      ],
+    },
+    threadID: {
+      propDefinition: [
+        discordWebhook,
+        "threadID",
+      ],
+    },
+    username: {
+      propDefinition: [
+        discordWebhook,
+        "username",
+      ],
+    },
+    avatarURL: {
+      propDefinition: [
+        discordWebhook,
+        "avatarURL",
+      ],
+    },
+    fileUrl: {
+      type: "string",
+      label: "File URL",
+      description:
+        "The URL of the file to attach. Must specify either File URL or File Path.",
```

Comment:
Bold File URL and File Path (see note on the File Path prop below)

Reply:
Updated